### PR TITLE
Answers for lesson 1

### PR DESCRIPTION
**Pull request by AnnaSolovykh**
Hi dear reviewer,

My name is Anna, and it is nice to meet you and start this wonderful Node journey with you!
I also want to apologize for the delay of the hometask... and to wish you a nice day ahead! :)

Kind regards, 
Anna Solovykh

### CODE REVIEW

File: 01-node-tutorial/answers/01-intro.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/01-intro.js:1]: 01-intro task complited.

Task one complited

File: 01-node-tutorial/answers/02-globals.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/02-globals.js:1]: 02-globals task complited.

Task two complited.

File: 01-node-tutorial/answers/03-modules.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/03-modules.js:1]: 03-module task complited.

Task three - seven complited.

File: 01-node-tutorial/answers/04-names.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/04-names.js:1]: Well done!

File: 01-node-tutorial/answers/05-utils.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/05-utils.js:1]: Well done!

File: 01-node-tutorial/answers/06-alternative-flavor.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/06-alternative-flavor.js:1]: Well done!

File: 01-node-tutorial/answers/07-mind-grenade.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/07-mind-grenade.js:1]: Well done!

File: 01-node-tutorial/answers/12-http.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/12-http.js:1]: Task 12 complited.

Remember to add a console log, to show the port on which the server is listening to

File: 01-node-tutorial/answers/prompter.js
Comment (leo-mutuku) [inline 01-node-tutorial/answers/prompter.js:1]: prompter task complited. 

Great for adding user-data.json and reading from it.

**Review by leo-mutuku — APPROVED** [review]

**Comment by leo-mutuku** [comment]
Well done! You have complited all the tasks correctly. 

#Notes
+ Commit each task singly and give a good commit message for example in task on intro, the commite message can be the file name and what is in the file like console logging in terminal.
+ For task 12 http, remember to console log the port on which the server is running on.

**Comment by AnnaSolovykh** [comment]
Thank you for the review! 
Ok, I will 🤝

**Pull request merged**